src/client.rs

```diff
@@ -3,55 +3,74 @@
 // Licensed under the MIT License, <LICENSE or http://opensource.org/licenses/MIT>.
 // This file may not be copied, modified, or distributed except according to those terms.
 
-use Packet;
-use futures::{Async, BoxFuture};
+use {WireError, tokio_proto as proto};
```

Comment:
imports are relative from the crate root; `WireError` wasn't already in scope. It's similar to how if you don't say `use std;` then you have to prefix paths with `::std::` rather than `std::`.